frontend/src/admin/components/dropdown-menu/dropdown-menu.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../../styles/main";
+
+.c-dropdown-menu {
+  position: relative;
+
+  button {
+    border: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .c-dropdown-menu-items {
+    display: none
+  }
+
+  &--active {
+    position: relative;
+
+    button {
```

Comment:
This is the exact same values as the `button {...}` specified 15 lines above.
Looking at the selectors, this styling would already be applied from that, so what's the need for specifying them again?

In general, you should try to only specify an attribute-value pair once. I.e. if you have several selectors needing `margin: 0`, then all selectors should point at the one place where `margin: 0` is specified instead of you writing `margin: 0` several times

Reply:
Could this be avoided via mixins:

@mixin button-no-style {
  border: none;
  margin: 0;
  padding: 0;
}

 And then include this on the different  buttons:
 
 button {
    @include button-no-style;
  }

Reply:
Definately. The less duplicated code, the better :)